src/oralb_ble/parser.py

```diff
@@ -341,19 +342,23 @@ async def async_poll(self, ble_device: BLEDevice) -> SensorUpdate:
             battery_payload = await client.read_gatt_char(battery_char)
             pressure_char = client.services.get_characteristic(CHARACTERISTIC_PRESSURE)
             pressure_payload = await client.read_gatt_char(pressure_char)
+            tb_pressure = ACTIVE_CONNECTION_PRESSURE.get(
+                pressure_payload[0], f"unknown pressure {pressure_payload[0]}"
+            )
+            self.update_sensor(
+                str(OralBSensor.PRESSURE), None, tb_pressure, None, "Pressure"
+            )
+            self.update_sensor(
+                str(OralBSensor.BATTERY_PERCENT),
+                Units.PERCENTAGE,
+                battery_payload[0],
+                SensorDeviceClass.BATTERY,
+                "Battery",
+            )
```

Comment:
Since these aren't expected to raise `BleakError` they should go in an `else:` block in the `try:` scope below

Reply:
Good call - thanks